playground/Elasticseach/Elasticseach.AppHost/Properties/launchSettings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "https": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "https://localhost:17196;http://localhost:15122",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "DOTNET_ENVIRONMENT": "Development",
+        "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL": "https://localhost:21146",
+        "DOTNET_RESOURCE_SERVICE_ENDPOINT_URL": "https://localhost:22194"
+      }
+    },
+    "http": {
```

Comment:
Add another launch profile in called `generate-manifest` (take a look at the other playground projects). Be sure to run generation on the manifest and add that file to the PR.